styles/components/_steps.scss

```diff
@@ -0,0 +1,47 @@
+.steps {
+  padding: 8px 15px;
+  list-style: none;
+  color: $brand-gray;
+  font-size: $small;
+
+  li {
+    position: relative;
+    display: inline-block;
+    margin: 0 1em;
+    width: 2em;
+    height: 2em;
+    padding: 2px 10px;
+    text-align: center;
+    background: $brand-gray-lt;
+    text-decoration: none;
+    border-radius: 1em;
+    border: $border-radius-base solid $brand-gray;
+  }
+
+  li > a { color: $brand-gray-dk; }
+
+  li::before{
+    content: '';
+    position: absolute;
+    top: 0.9em;
+    left: -4em;
```

Comment:
I mean, why? I don't really get it. And imo, I'd prefer `rem` because `em` has its side-effects (like the parent's `em` can affect it since depends on it)